tasks/verifySpokePool.ts

```diff
@@ -64,7 +64,9 @@ task("verify-spokepool", "Verify the configuration of a deployed SpokePool")
     console.log("SpokePool.fillDeadlineBuffer()".padEnd(TEXT_PADDING) + ": " + fillDeadlineBuffer);
 
     const wrappedNative = bytes32ToAddress(results.wrappedNativeToken);
-    const wrappedNativeSymbol = spokeChainId === POLYGON ? "WMATIC" : "WETH";
+    const nativeTokenSymbol = PRODUCTION_NETWORKS[spokeChainId].nativeToken;
+    const wrappedPrefix = "W";
+    const wrappedNativeSymbol = `${wrappedPrefix}${nativeTokenSymbol}`;
```

Comment:
I'm not sure this will always be the case. Should we be explicit here?

Reply:
The idea with this approach is that we avoid adding a new chain for each custom gas token. I think we can go with this approach and revisit when there's a case that doesn't fit with this logic. wdyt?

Reply:
@james-a-morris My thinking is that it's worth proceeding with until we find the exception to the rule, and then re-evaluate in that case. The cost of adapting it later is quite low, and then we'd have the advantage of knowing what the alternatives scenario(s) is/are.